src/BlazorUI/Bit.BlazorUI/Components/Inputs/Rating/BitRating.scss

```
@@ -1,9 +1,23 @@
-﻿@import "../../../Styles/functions.scss";
+@import "../../../Styles/functions.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)